README.md

```diff
@@ -17,6 +17,24 @@ Reader.open('/usr/local/share/GeoIP/GeoIP2-City.mmdb').then(reader => {
 });
 
 ```
+### Anonymous IP Database Example
+
+```
+const reader = require('GeoIP2-node').Reader;
+
+Reader.open('/usr/local/share/GeoIP/GeoIP2-Anonymous-IP.mmdb').then(reader => {
+  const response = reader.anonymousIP('85.25.43.84');
+
+  console.log(response.is_anonymous); // true
+  console.log(response.is_anonymous_vpn); // false
+  console.log(response.is_hosting_provider); // false
+  console.log(response.is_public_proxy); // false
+  console.log(response.is_tor_exit_node); // true
```

Comment:
I'm getting different results for `85.25.43.84`. It might be confusing if someone were to try the sample IP for testing, and get different results.
What's returned:
```
  console.log(response.is_anonymous); // true
  console.log(response.is_anonymous_vpn); // undefined
  console.log(response.is_hosting_provider); // true
  console.log(response.is_public_proxy); // undefined
  console.log(response.is_tor_exit_node); // undefined
```

Reply:
Does this even matter?

Reply:
Also, should the `undefined` values be `false`?

Reply:
For the other GeoIP2 libraries, we do set them to false, both here and in `traits` for City, etc.

Reply:
I've made the fix for the anonymous db values in here https://github.com/maxmind/GeoIP2-node/pull/7/commits/0e79594c7caf5948b64c1e8d8c76e006a703d586.  I'll make a separate PR for fixing other the other boolean values found in other props

Reply:
Here's the fix for the rest of boolean values: https://github.com/maxmind/GeoIP2-node/pull/8/commits/3a4ec6fb10d616dd654088b550a3a8bfbf2a417d

Reply:
@oschwald Do the other libraries return false for City database  `traits.is_legitimate_proxy` queries?  I ask because this is only available in Enterprise

Reply:
Ah, they do.